madcars/players/pytorch_main.py

```diff
@@ -207,6 +219,8 @@ def compute_td_loss(states, actions, rewards, next_states, is_done, gamma=0.99,
             states.append(state)
             actions.append(choice)
             rewards.append(0)
+            for i in range(len(rewards)):
+                rewards[i] -= 10  # penalize for time spent not winning
             dones.append(False)
             # update epsilon:
             epsilon = min(epsilon * epsilon_decay, 0.1)
```

Comment:
(Should there be max?)
There is no reason to decay epsilon in local memory because after 10 games it will be reset. I believe you never even get to 0.1.

Reply:
there should be **max**, right, will fix.

let's leave it "as is" for now. in the long run we should be able to benefit from it (right?)

Reply:
Okay (may be)